server/models/annotation.py

```diff
@@ -49,7 +49,8 @@ def createAnnotation(self, study, image, user):
             'startTime': None,
             'stopTime': None,
             'status': None,
-            'annotations': None
+            'responses': {},
+            'markups': {},
```

Comment:
https://github.com/ImageMarkup/isic-archive/wiki/Annotation describes these as lists, but it looks like they're mappings. Should the wiki be clarified, including the example JSON"

Reply:
They should be mappings, this code is incorrect. However, what's being set here is an incomplete annotation. Once an annotation is submitted, the whole field is overwritten, so this code doesn't really matter. However, I was thinking of initializing these fields as empty containers instead of `None` just to make aggregation development more convenient.

Reply:
Initializing them as empty containers makes sense. I meant we should clarify in the documentation whether they are of `list` or `dict` type. Currently the example on the wiki is invalid JSON that mixes list and object formats, e.g.:
```
"responses": [
    "Classify this lesion as benign or malignant": "Benign",
    "Enter any other clinical observations": "atypical nevus"
],
```

Reply:
Got it, that's a typo.